lib/Importer.js

```diff
@@ -410,6 +410,9 @@ export default class Importer {
         this.pathToCurrentFile,
         this.workingDirectory,
       ),
+      importStatement: jsModule
```

Comment:
Instead of adding another key, I think you should just change `displayName` to be the import statement.